docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -2,14 +2,15 @@
 
 The buoyancy option selects how buoyancy is treated. There are currently three options:
 
-1. No buoyancy (and no gravity).
+1. No buoyancy (and no gravity). This is the default setting.
 2. Evolve buoyancy as a tracer.
 3. _Seawater buoyancy_: evolve temperature ``T`` and salinity ``S`` as tracers with a value for the gravitational
-   acceleration ``g`` and an equation of state of your choosing. This is the default setting.
+   acceleration ``g`` and an equation of state of your choosing.
 
 ## No buoyancy
 
-To turn off buoyancy (and gravity) simply pass `buoyancy = nothing` to the model constructor.
+To turn off buoyancy (and gravity) you can simply create a model without explocitly passing the
```

Comment:
```suggestion
To turn off buoyancy (and gravity) you can simply create a model without explicitly passing the
```

Reply:
I can commit all suggestions if you agree with them

Reply:
Thanks, I'd rather try to figure out what's going on first though. It's super weird, I just refreshed the page and the commit button appeared for some of your suggestions, but not other. So maybe github is having issues?

Reply:
Refresh again perhaps!!

Reply:
I've been doing that. Also reading up on it. I think github is just facing a but of instability. I'll comment on the suggestions that I agree with and then if you can commit those if you'd like. Otherwise I'll try it again tomorrow morning. Thank you in advance :)

I definitely agree with this suggestion!